cdap-ui/app/cdap/components/PipelineScheduler/Store/index.js

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import {defaultAction} from 'services/helpers';
+import {createStore, compose} from 'redux';
+
+const INTERVAL_OPTIONS = [
+  {
+    id: '5min',
```

Comment:
If possible have a map of these `id`s. I see `switch` statements in multiple places using the same id. Would be useful if we specify in a constant and use that constant in all the `switch` `case` statements.